content/en/docs/contribute/style/style-guide.md

```diff
@@ -380,6 +385,24 @@ The output is:
 
 {{< kat-button >}}
 
+### Downloadable code blocks
+
+Use {{</* codenew */>}} to make code samples (eg manifests) downloadable.
+Pages that provide ready-to-run sample manifests should put them in respective localized folder, (eg) content/en/examples/application/deployment.yaml (for English localization) and reference the downloadable code sample directly, using the {{</* codenew */>}} shortcode.
+
+eg: {{</* codenew file="application/deployment.yaml" */>}}
+
```

Comment:
Line 390: Could you remove the `eg` in (eg manifests)?

Line 393 Could you remove the `eg:` and add For example:
Then, enclose the sample in triple back ticks. See the note shortcode example on the page (line 282):

 ```
    {{</* note */>}}
    No need to include a prefix; the shortcode automatically provides one. (Note:, Caution:, etc.)
    {{</* /note */>}}
```

Reply:
updated.